src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -1046,14 +1034,56 @@ const schema = {
               'Medical Treatment Record - Government Facility',
               'Medical Treatment Record - Non-Government Facility',
               'DD214',
-              'Military Personnel Record',
-              'Decorations/Awards/Medal Citations',
               'Other Correspondence',
               'Buddy/Lay Statement'
             ]
           }
         }
-
+      }
+    },
+    unemployabilityAttachments: {
```

Comment:
@dmccurdybah @aaronjlech (@abhijitvk as FYI) sorry I didn't get a chance to review this PR before it was merged. I think the current way that `vets-api` parses the 526 form data to look for attachments is just to look at everything in the `attachments` key. So I think that any data submitted under an `unemployabilityAttachments` property will get ignored by `vets-api` and not actually attach to the form / pdf that's submitted to vbms. Can you provide some context on how / if this key is handled by `vets-api`?

Though not part of this PR, same goes for `employmentRequestAttachments`, `secondaryAttachment`, `completedFormAttachments`, and `privateMedicalRecordAttachments`. And any other attachment property that isn't called `attachments` 😛 

What probably needs to happen to enable this is:
1. Duplicate `attachments` property into `definitions`
2. Update the `attachments` property to `{ $ref: '#/definitions/vaTreatmentCenterAddress' }`
3. Delete all the other `___attachments` properties
4. In `vets-website`, update the page schemas for upload pages that need to have a restricted subset of selectable `attachmentId`s - they can use the `attachments` definition and modify from there.

Apologies if I've misunderstood how the api currently handles our attachments - if there's a plan to parse the submitted form data for all of these attachment keys, let me know.